packages/grafana-llm-app/src/components/AppConfig/LLMConfig.tsx

```diff
@@ -221,6 +235,14 @@ export function LLMConfig({
             </Card.Figure>
           </Card>
         </div>
+        {(llmOption === 'test' || previousOpenAIProvider === 'test') && (
```

Comment:
Maybe display this when `process.env.NODE_ENV === 'development'` so that it isn't required to know about provisioning it to use test mode in dev?

Reply:
Nice idea :+1: It still feels weird to me using environment variables in frontend code, I forget that it works!